helloworld.py

```diff
@@ -1,4 +1,4 @@
-print("hellooo Ron1111")
+print("hellooo Ron11222333 ve ze ba bash1")
 
 
 
```

Comment:
1- No more hebrew :)
2- Prefer single quotes (`'my single quote string'`) over double quotes (`"this is less pythonic"`)

I'll approve this time :)